setup.py

```diff
@@ -13,7 +11,8 @@
       author='Tom Aldcroft',
       description='Sparkles ACA review package',
       author_email='taldcroft@cfa.harvard.edu',
-      version=__version__,
+      use_scm_version=True,
+      setup_requires=['setuptools_scm', 'setuptools_scm_git_archive'],
```

Comment:
@javierggt - these two `setup_requires` are sufficient to enable the current build process for Windows:
- Download zip file corresponding to a tagged release
- Unzip
- `python setup.py install` (into an SVN-controlled distribution that is then synced to user laptops)
This will auto-download those from pypi for the build step, so a network is required, but that is not a problem for building.